### ISSUE TITLE
DJ012 triggers one violation per missplaced database field. Could be grouped as one single one?

### ISSUE THREAD

**Issue by EricDepagne** [astral-sh/ruff#5465]
Hi all.
I' m using Ruff with a django project.
Whenever DJ012 is triggered (which in my case  is when Meta Class is before the database fields fields), each violation of the same rule in the same model is counted as one.
Would it be possible to group them all tgether, as they are technically the same, so to report only one?

Here' s a short example:
```
class Activity(models.Model):
    """
    Define an activity
    """

    class Meta:
        verbose_name_plural = "Activities"
    name = models.CharField(max_length=50, blank=True)
    temp = ArrayField(
        models.PositiveSmallIntegerField(blank=True, default=0, null=True),
        default=list,
    )

```
will trigger two violations, as they are two fields after the class Meta that will trigger the rule.

**Comment by charliermarsh** [comment]
Yeah, I think this would be better behavior. Nice idea.

**Comment by EricDepagne** [comment]
Wow! that' s an amazing time to fix a very minor issue that I only reported because I thought it might be a good idea!
Thank you so much!
